### ISSUE TITLE
[BUG] Table missing certain ARIA roles that must contain particular children

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#117]
**Describe the bug**
The admiralty-table-header has a role="rowgroup" with children with the role="columnheader" when it's expecting children with a role="row". Certain ARIA roles must contain particular children table > rowgroup > row > cell | coulmnheader | rowheader.

**To Reproduce**
Inspect the admiralty-table component and check the roles of each child.
Or run the Axe-core automated tests on a page containing the admiralty-table component

**Expected behavior**
There needs to be a admiralty-table-row inserted between admiralty-table-header and admiralty-table-header-cell.

**Screenshots**
![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/e5a05916-ff8a-4602-8645-ab195aa6c9ac)

**Additional context**
Using a native HTML table element whenever possible is strongly encouraged. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table
